app/views/admin_posts/_admin_post.html.erb

```diff
@@ -1,7 +1,7 @@
 <% # expects "admin_post" %>
-<div <% if rtl? %>dir="rtl"<% end %> class="header">
+<div <% if post_rtl?(admin_post) %>dir="rtl"<% end %> class="header">
   <h3 class="heading">
-    <%= link_to admin_post.title.html_safe, admin_post %>
+   <%= link_to admin_post.title.html_safe, admin_post %>
```

Comment:
Minor thing: can you indent this one more space?

Reply:
Ah, yes, I think my IDE's autoformatter got confused.